opencog/atomspace/CountTruthValue.cc

```diff
@@ -34,20 +34,20 @@
 
 using namespace opencog;
 
-CountTruthValue::CountTruthValue(strength_t m, confidence_t n, count_t c)
+CountTruthValue::CountTruthValue(strength_t m, confidence_t n, count_t c):TruthValue(COUNT_TRUTH_VALUE)
```

Comment:
Please don't do this. This makes the TV larger than 8 bytes. There is no reason to store a compile time constant as if it were some dynamic, run-time variable.